docs/articles/dockerfile_best-practices.md

```diff
@@ -153,9 +153,12 @@ maintainers. If you know there’s a particular package, `foo`, that needs to be
 updated, use `apt-get install -y foo` and it will update automatically.
 
 * Do write instructions like:
-
-    RUN apt-get update && apt-get install -y package-bar package-foo package-baz
-
+```
+    RUN apt-get update && apt-get install -y \
+        package-bar \
+        package-foo \
+        package-baz
+```
```

Comment:
I tried tab indent but it didn't look right in GitHub preview. Ie. It turned into a single line with slashes. Would it render properly when the site is generated?